### PR TITLE
Add unit test for swssnet.h

### PR DESCRIPTION
**Pull request by qiluo-msft**

### CODE REVIEW

File: tests/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+CFLAGS_GTEST =
+LDADD_GTEST =
+
+tests_SOURCES = swssnet_ut.cpp
+
+tests_CFLAGS = $(DBGFLAGS) $(AM_CFLAGS) $(CFLAGS_COMMON) $(CFLAGS_GTEST) $(CFLAGS_SAI)
+tests_CPPFLAGS = $(DBGFLAGS) $(AM_CFLAGS) $(CFLAGS_COMMON) $(CFLAGS_GTEST) $(CFLAGS_SAI)
+tests_LDADD = $(LDADD_GTEST) -lnl-genl-3 -lhiredis -lhiredis -lpthread \
+        -lswsscommon -lswsscommon -lgtest -lgtest_main
+
```
Comment (stcheng) [inline tests/Makefile.am]: empty spaces

File: tests/swssnet_ut.cpp [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    IpPrefix ip("2001:4898:f0:f153:357c:77b2:49c9:627c/27");
+    sai_ip_prefix_t dst;
+    copy(dst, ip);
+    EXPECT_EQ(dst.addr_family, SAI_IP_ADDR_FAMILY_IPV6);
+    
+    char buf[INET6_ADDRSTRLEN];
+    inet_ntop(AF_INET6, dst.addr.ip6, buf, INET6_ADDRSTRLEN);
+    EXPECT_STREQ(buf, "2001:4898:f0:f153:357c:77b2:49c9:627c");
+    inet_ntop(AF_INET6, dst.mask.ip6, buf, INET6_ADDRSTRLEN);
+    printf("%s\n", buf);
```
Comment (stcheng) [inline tests/swssnet_ut.cpp]: remove this?

File: tests/swssnet_ut.cpp [hunk 3]
```diff
@@ -0,0 +1,64 @@
+#include <arpa/inet.h>
+#include <gtest/gtest.h>
+#include <string>
+#include <iostream>
+#include "swssnet.h"
+
+using namespace std;
+using namespace swss;
+
+TEST(IpAddress, copy)
```
Comment (stcheng) [inline tests/swssnet_ut.cpp]: better to use TEST(module_name, submodule_name) format, so it is better to use filter to test all swssnet functionalities

Comment (stcheng) [inline tests/swssnet_ut.cpp]: same as below

File: tests/Makefile.am (hunk 1 above)
Comment (qiluo-msft) [inline tests/Makefile.am]: Resolved.

File: tests/swssnet_ut.cpp (hunk 2 above)
Comment (qiluo-msft) [inline tests/swssnet_ut.cpp]: Resolved.

File: tests/swssnet_ut.cpp (hunk 3 above)
Comment (qiluo-msft) [inline tests/swssnet_ut.cpp]: sure. prefer keep test cases fine granularity because of easy variable naming.

Comment (stcheng) [inline tests/swssnet_ut.cpp]: i believe the name can be the same. not sure. you could check.

Comment (qiluo-msft) [inline tests/swssnet_ut.cpp]: I mean variable names inside each test case.
Resolved.

**Pull request merged**